application/views/questionAdministration/textElements.php

```diff
@@ -129,7 +145,7 @@
                         "questionI10N[$lang][script]",
                         $question->questionl10ns[$lang]->script,
                         [
-                            'id' => CHtml::getIdByName("questionI10N[{$lang}][script]"),
+                            'id' => $scriptFieldId,
```

Comment:
We use `<?= CHtml::encode($scriptFieldId); ?>` , unsure it's needed, but we must do same for both.